pkg/apply/solver/solver_test.go

```diff
@@ -1494,11 +1385,11 @@ func TestTaskQueueBuilder_PruneBuild(t *testing.T) {
 				Pruner:    pruner,
 				Mapper:    mapper,
 				InvClient: fakeInvClient,
+				Inventory: inventoryObj,
 				Collector: vCollector,
 			}
 			taskContext := taskrunner.NewTaskContext(t.Context(), nil, nil)
-			tq := tqb.WithInventory(invInfo).
-				WithPruneObjects(tc.pruneObjs).
+			tq := tqb.WithPruneObjects(tc.pruneObjs).
```

Comment:
Doesn't look new, and you don't need to fix it here, but it's kinda weird that the TaskQueueBuilder takes some public fields and others have to be set with the With- methods...

Reply:
looks like you fixed this. thanks!